src/components/MoviesList/component.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import MovieItem from 'Components/MovieItem'
+import { Col } from 'antd'
+
+const MoviesList = ({ moviesIDs, moviesData }) => (
+  moviesIDs && moviesIDs.map((item) => {
+    const {
+      id, poster_path: poster, name, title, overview
+    } = moviesData[item]
```

Comment:
create `moviesSelector` and use `pick` function from `lodash` to retrieve movies

Reply:
don't forget to create moviesSelector and use pick function from lodash to retrieve movies